libs/angular/src/platform/services/logging-error-handler.ts

```diff
@@ -1,14 +1,18 @@
-import { ErrorHandler, Injectable } from "@angular/core";
+import { ErrorHandler, Injectable, Injector, inject } from "@angular/core";
 
 import { LogService } from "@bitwarden/common/platform/abstractions/log.service";
 
 @Injectable()
 export class LoggingErrorHandler extends ErrorHandler {
-  constructor(private readonly logService: LogService) {
-    super();
-  }
+  /**
+   * When injecting services into an `ErrorHandler`, we must use the `Injector` manually to avoid circular dependency errors.
+   *
+   * https://stackoverflow.com/a/57115053
+   */
+  private injector = inject(Injector);
 
   override handleError(error: any): void {
-    this.logService.error(error);
+    const logService = this.injector.get(LogService);
+    logService.error(error);
```

Comment:
Let's wrap this in a try/catch and fall back to `super` as @justindbaur suggested in slack.
```suggestion
  override handleError(error: any): void {
    try {
      const logService = this.injector.get(LogService, null);
      logService.error(error);
    } catch {
      super.handleError(error);
    }
  }
```

Reply:
LGTM, thanks. Committed in [0c25ca9](https://github.com/bitwarden/clients/pull/8573/commits/0c25ca98a7bc7211177aedbb8d058e0087bc2952)